package.json

```diff
@@ -272,7 +272,7 @@
     "@types/cors": "^2.8.5",
     "@types/express": "4.17.3",
```

Comment:
May want to also update the types to match?

Reply:
Oh interesting - there is no type versions for 4.19.2:

https://www.npmjs.com/package/@types/express?activeTab=versions

I'll bump it to the latest, compatible version which is 4.17.21.